install/FullInstall/fullInstall.sql

```diff
@@ -902,3 +902,21 @@ UPDATE `wD_Misc` SET `value` = '161' WHERE `name` = 'Version';
 
 ALTER TABLE `wD_Games` ADD COLUMN `playerTypes` enum('Members', 'Mixed', 'MemberVsBots') DEFAULT 'Members' NOT NULL;
 ALTER TABLE `wD_Backup_Games` ADD COLUMN `playerTypes` enum('Members', 'Mixed', 'MemberVsBots') DEFAULT 'Members' NOT NULL;
+
+ALTER TABLE `wD_Games`
+ADD COLUMN `nextPhaseMinutes` int(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `phaseMinutes`,
+ADD COLUMN `phaseSwitchPeriod` int(10) DEFAULT -1 AFTER `nextPhaseMinutes`,
+ADD COLUMN `startTime` int(10) UNSIGNED DEFAULT NULL;
+
+UPDATE `wD_Games`
+SET `nextPhaseMinutes` = `phaseMinutes`
+WHERE `nextPhaseMinutes` = 0;
+
+ALTER TABLE `wD_Backup_Games`
+ADD COLUMN `nextPhaseMinutes` int(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `phaseMinutes`,
+ADD COLUMN `phaseSwitchPeriod` int(10) DEFAULT -1 AFTER `nextPhaseMinutes`,
+ADD COLUMN `startTime` int(10) UNSIGNED DEFAULT NULL;
+
+UPDATE `wD_Backup_Games`
+SET `nextPhaseMinutes` = `phaseMinutes`
+WHERE `nextPhaseMinutes` = 0;
```

Comment:
Same sql update for version needed here.

Reply:
Done